src/contexts/language/languageContextBinding.ts

```diff
@@ -7,6 +7,8 @@ import { LanguageContextFactory, Types } from '../../types';
 import { bindProjectComponentContext } from '../projectComponent/projectComponentContextBinding';
 import { JavaPackageInspector } from '../../inspectors/package/JavaPackageInspector';
 import { JavaComponentDetector } from '../../detectors/Java/JavaComponentDetector';
+import { GolangPackageInspector } from '../../inspectors/package/GolangPackageInspector';
```

Comment:
nitpick but maybe should standardize whether to use `Go` or `Golang`

Reply:
What is preferred in Golang community? :)
IMHO I think Golang would be better. What do you think? @ronaudinho

Reply:
AFAIK `Go` is preferred (personal, most talks I've seen use `Go` instead of `Golang`, and [this](https://golang.org/doc/faq#go_or_golang)), but I'm okay both.

Reply:
Ok, so can you change it everywhere from Golang to Go? :)

Reply:
done